cli/cmd/root.go

```diff
@@ -98,6 +99,7 @@ var RootCmd = &cobra.Command{
 }
 
 func init() {
+	RootCmd.PersistentFlags().StringVarP(&kubeNamespace, "namespace", "n", getDefaultNamespace(), "Namespace for specified resource")
```

Comment:
It's not clear to me how this interacts with the namespace flag created by specific commands.  For example, the `edges` command has a namespace flag which takes its default value from its options struct which is populated by the `kubeNamespace` variable which is populated here.  😵 

I think it would be more straightforward to simply populate the kubeNamespace variable directly here: `kubeNamespace = getDefaultNamespace()` and then allow individual commands to define their own namespace flag.

Reply:
That makes a lot of sense, and it is something I was worried about when I decided to go with the global namespace flag.

Think this makes sense though, I thought for some reason initialising the variable in `init` would be frowned upon a bit more than the flag. Will change it asap!